release-notes.md

```diff
@@ -8,6 +8,24 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## February 18, 2020
+
+The following updates were made in this release:
+
+### iOS Identity 2.2.0
+
+* Report extension details to Mobile Core for improved logging and Griffon support.
+* Added APIs to support `completionHandler` callback that is available in iOS ACPCore version 2.5.0 on the following APIs:
+
+  * `appendToURL:witCompletionHandler`
+  * `getUrlVariablesWithCompletionHandler`
+  * `getIdentifiersWithCompletionHandler`
+  * `getExperienceCloudIdWithCompletionHandler`
+
+  When the `completionHandler` is used, and you are retrieving the Mobile SDK values, the timeout value is 500ms; if the operation times out or is not successful, an `NSError` is returned.
+
+Released with ACPCore version 2.5.0.
```

Comment:
Please also include these release notes in https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/blob/master/using-mobile-extensions/mobile-core/mobile-core-release-notes.md and update the iOS APIs documentation with the new methods

Reply:
@emdobrin I'm updating the Identity API reference in another PR.